lib/eslint.js

```diff
@@ -527,7 +527,7 @@ function getRuleReplacementMessage(ruleId) {
     if (ruleId in replacements.rules) {
         const newRules = replacements.rules[ruleId];
 
-        return "Rule \'" + ruleId + "\' was removed and replaced by: " + newRules.join(", ");
+        return `Rule \'${ruleId}\' was removed and replaced by: ${newRules.join(", ")}`;
```

Comment:
Good catch! Do we not have no-useless-escape turned on? On my phone now, but will look into this later.